pennylane/gradients/parameter_shift.py

```diff
@@ -954,12 +963,10 @@ def param_shift(
 
     .. warning::
 
-        Note that using parameter broadcasting via ``broadcast=True`` is not supported for tapes
-        with multiple return values or for evaluations with shot vectors.
-        As the option ``broadcast=True`` adds a broadcasting dimension, it is not compatible
-        with circuits that already are broadcasted.
-        Finally, operations with trainable parameters are required to support broadcasting.
-        One way of checking this is the `Attribute` `supports_broadcasting`:
+        Note that as the option ``broadcast=True`` adds a broadcasting dimension, it is not compatible
+        with circuits that are already broadcasted.
+        In addition, operations with trainable parameters are required to support broadcasting.
+        One way to check this is through the ``supports_broadcasting`` attribute:

```

Comment:
Is the information that `"parameter broadcasting via ``broadcast=True`` is not supported for tapes with multiple return values or for evaluations with shot vectors."` out of date in the 0.37 release?

Reply:
Yes, [this PR](https://github.com/PennyLaneAI/pennylane/pull/5667) addressed exactly that :)